src/api/app/models/group.rb

```diff
@@ -5,6 +5,7 @@
 # Groups have an arbitrary number of roles and users assigned to them.
 #
 class Group < ApplicationRecord
+  has_one :staging_workflow, foreign_key: :managers_id, dependent: :nullify
```

Comment:
Why would we need this?

Reply:
I don't think we will ever need this from the Group direction. At least not now...

Reply:
Probably... I'm fine dropping it but let's wait for @mdeniz

Reply:
Dropped ...

Reply:
But if the admin removes the group? what happen with the staging workflow? it will have an id for a group that doesn't exist

Reply:
Ok I will add it again ...

Reply:
Sorry for the confusion...

Reply:
One concern I have though is that this will invalidate the staging workflow because the managers_group is validated to be present (eventually it will succeed because it is done with raw SQL in rails). Thoughts @mdeniz

Reply:
Sure, it will invalidate any Staging Workflow that uses that group as managers group... So, either you add a confirm box at deletion of the group if it have any staging workflow assigned or you don't care about it and let it as it is.

Reply:
Yo, that was just something to think about it. We can do it later on, I don't think this will be much of a use case in real life (I don't think sb will delete ``factory-managers`` group)

Reply:
factory-managers is an important one... but any other groups not related to the distribution can be deleted without noticing it

Reply:
> factory-managers is an important one... but any other groups not related to the distribution can be deleted without noticing it

Yep sure, just meant for the 'skateboard' this is not going to happen because there will be only factory-managers for now. So this is something we might address in a later sprint (or not at all :thinking: )